packages/objects-horizontal-list/_horizontal-list.scss

```diff
@@ -63,7 +69,11 @@ $horizontal-list-delimiter: ',\00A0' !default;
 ///
 .#{$scales-namespace}#{$scales-objects-class-namespace}HorizontalList--delimited {
   > li {
+    margin-right: 0;
+
     &::after {
+      margin-left: $horizontal-list-spacing / 2;
```

Comment:
I'm not sure what this is doing.

Reply:
That's to split the spacing setting in half when the delimiter is set. So half the space before the | and half after.